cluster/images/etcd-version-monitor/README.md

```diff
@@ -0,0 +1,30 @@
+# etcd-version-monitor
+
+This is a tool for exporting metrics exposed by etcd endpoints that are not
+in prometheus metrics format. The exported metrics would be in prometheus
+metrics format and would be exposed by this tool at localhost:9101/metrics
+http endpoint.
+
+**RUNNING THE TOOL**
+
+To run this tool as a binary:
+- make
+- ./bin/etcd-version-monitor
```

Comment:
This will not work if the other binaries are not in correct places. I wouldn't recommend this method.